modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -43,42 +43,34 @@ Attempting to register openSUSE Leap 15 or SLE 15 systems using Python 2 boot
 
 == Create a Bootstrap Script
 
-This procedure describes how to generate a bootstrap script.
+You can use the {productname} {webui} to create an editable bootstrap script.
 
 .Procedure: Creating a Bootstrap Script
 . In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bootstrap Script].
 . In the [guimenu]``SUSE Manager Configuration - Bootstrap`` dialog, uncheck the [guimenu]``Bootstrap using Salt`` checkbox if you are installing a traditional client.
 For Salt clients, leave it checked.
-Use default settings and click the btn:[Update] button.
-+
+. The required fields are pre-populated with values derived from previous installation steps.
+For details on each setting, see xref:reference:admin:bootstrap-script.adoc[].
+. Click btn:[Update] create the script.
+. The bootstrap script is generated and stored on the server in the [path]``/srv/www/htdocs/pub/bootstrap`` directory.
+Alternatively, you can access the bootstrap script over HTTPS.
+Replace ``example.com`` with the host name of your {productname} Server:
+----
+https://EXAMPLE.COM/pub/bootstrap/bootstrap.sh
+----
 
-image::mgr_configuration_bootstrap_trad.png[scaledwidth=80%]
-+
 
-[WARNING]
-.Using SSL
+[IMPORTANT]
 ====
-Unchecking [guimenu]``Enable SSL`` in the {webui} or setting `USING_SSL=0` in the bootstrap script is not recommended.
-If you disable SSL nevertheless you will need to manage custom CA certificates to be able to run the registration process successfully.
+Do not disable SSL in your bootstrap script.
+Ensure that [guimenu]``Enable SSL`` is checked in the {webui}, or that the setting `USING_SSL=1` exists in the bootstrap script.
+If you disable SSL, the registration process requires custom SSL certificates.
```

Comment:
Maybe, adding an xref to the section about creating custom SSL certificates here?  IIRC, you recently work on such a section :)

Reply:
Good idea.